src/core/tests/Test_AzL3TdnfPackageManager.py

```diff
@@ -165,386 +165,6 @@ def test_do_processes_require_restart(self):
         self.assertTrue(package_manager.is_reboot_pending())  # returns true because the safe default if a failure occurs is 'true'
         package_manager.do_processes_require_restart = backup_do_processes_require_restart
```

Comment:
Are the tests that remain here truly AzL3 specific?

Like the immediate one above -- test_do_processes_require_restart(self):

Reply:
Also are utility functions and mocks:
- correctly still required in each test class? i.e. no dead code
- duplicated unnecessarily?